File: benchmark/workloads/csharp/fibonacci.cs
static long Fibonacci(long value)
{
    if (value < 2)
    {
        return value;
    }
    return Fibonacci(value - 1) + Fibonacci(value - 2);
}

long total = 0;
for (var index = 0; index < 30; index++)
{
    total += Fibonacci(28);
}
Console.WriteLine(total);

File: benchmark/workloads/c/fibonacci.c
#include <inttypes.h>
#include <stdint.h>
#include <stdio.h>

static uint64_t fibonacci(uint64_t value) {
    if (value < 2) {
        return value;
    }
    return fibonacci(value - 1) + fibonacci(value - 2);
}

int main(void) {
    uint64_t total = 0;
    for (uint64_t index = 0; index < 30; ++index) {
        total += fibonacci(28);
    }
    printf("%" PRIu64 "\n", total);
    return 0;
}
